src/overlays/fbdemos/ovl_fbdemo_triforce/z_fbdemo_triforce.h

```diff
@@ -3,8 +3,29 @@
 
 #include "global.h"
 
+typedef enum {
+    /* 0 */ STATE_NONE,
+    /* 1 */ STATE_SPIRAL_IN_SLOW,
+    /* 2 */ STATE_SPIRAL_IN_FAST,
+    /* 3 */ STATE_SPIRAL_OUT_SLOW,
+    /* 4 */ STATE_SPIRAL_OUT_FAST,
+} FbTriforceState;
+
+typedef enum {
+    /* 0 */ TYPE_NONE,
+    /* 1 */ TYPE_TRANSPARENT_TRIFORCE,
+    /* 2 */ TYPE_FILLED_TRIFORCE,
+} TriforceFadeType;
```

Comment:
```suggestion
    /* 2 */ TYPE_FILLED_TRIFORCE
} TriforceFadeType;
```
Could you remove the last comma off the enum? same with `FbTriforceState`